packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -1,81 +1,59 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '../utilities/style/display';
 import {
-    bodyFontColor,
-    borderHoverColor,
-    borderWidth,
     controlHeight,
     iconSize,
-    linkFont,
+    linkDisabledFontColor,
+    mediumPadding,
     placeholderFontColor
 } from '../theme-provider/design-tokens';
-import { focusVisible } from '../utilities/style/focus';
+import { styles as anchorStyles } from '../anchor/styles';
 
 export const styles = css`
-    ${display('inline-flex')}
+    ${anchorStyles}
```

Comment:
I considered forking the breadcrumb item template and changing it to use a `nimble-anchor` rather than a native anchor, but just reusing the anchor's styles does about the same thing without introducing extra elements in the DOM.

Most of the styles I removed from this file were redundant with the anchor styles.

Reply:
Scanning quickly it also looks like there are styles in the anchor pattern that might not be relevant to breadcrumb. So how we want to avoid duplicated / overrides in shared styles we should also avoid dead styles / styles that don't apply.

In the other comment I stated I'll take a look at the styles in a separate branch so I'll take a pass at evaluating the dead styles as well.

Reply:
As we discussed, I am no longer sharing styles with the anchor. I'm just copying the relevant parts.